### ISSUE TITLE
/views does not list all the views

### ISSUE THREAD

**Issue by brkattk** [oddnetworks/oddworks#211]
I'm hitting `content.oddworks.io/v2/views` and it only seems to be returning views with UUID ids, not custom slug ids.

**Comment by aljachimiak** [comment]
@brkattk 

The `catalog-list-controller` has a limit on it to prevent getting too much stuff:

````javascript
get(req, res, next) {
		const channelId = (req.identity.channel || {}).id;
		const limit = parseInt(req.query.limit, 10) || 10;
````

Is it possible that you are only retrieving _some_ of the views, rather than _all_ of them?

-Al

**Comment by brkattk** [comment]
Possibly. Although if limits filters or sorts are present it should be noted in top level request meta. I don't remember if that was being noted

**Comment by aljachimiak** [comment]
>Although if limits filters or sorts are present it should be noted in top level request meta

I agree.  Just to be super clear.  I'm only aware of officially supported `page[limit]`, `page[offset]`, and [sort] queries on the `/{type}s/:id/relationships/:relationshipKey endpoint.

This piece of [documentation seems to be incorrect here](http://code.oddnetworks.com/docs/#views_get).

This piece of [documentation seems to be correct](http://code.oddnetworks.com/docs/#views__viewId__relationships__relationshipKey__get).

IIRC - @Kiixauth mentioned in passing that this default limit was put here to protect the db from large responses.

I"m not sure that the default limit would be noted in a response.

**Comment by brkattk** [comment]
Gotcha. I think eventually those docs are correct. We just aren't there
yet. If any limit, even default, is placed on a list endpoint, then we
should show it in the response meta (so top level)

**Comment by brkattk** [comment]
Any "list" endpoint should have the following filter abilities:

- offset
- limit
- sort

Any time we have any of these present, even defaults, the top-level meta data must reflect the values used.

ex. 
```json
{
  "data": [ 
    // ...
  ],
  "meta": {
    "total": 15,
    "limit": 10
  }
}
```

Otherwise, it's not really possible to know what is happening when you get a result and the expected items aren't in the list.

**Comment by aljachimiak** [comment]
This can be closed after an issue is created for the above comment.

**Comment by kixxauth** [comment]
This issue (listing views) is probably blocked by this related paging issue: https://github.com/oddnetworks/oddworks/issues/282
